src/com/owncloud/android/authentication/PinCheck.java

```diff
@@ -0,0 +1,30 @@
+package com.owncloud.android.authentication;
+
+import android.app.Activity;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import com.owncloud.android.MainApp;
+
+public class PinCheck extends Activity {
```

Comment:
This should not be an Activity; it's never started, it only holds a couple of static methods... it should be a Utils class, or would be possible just moving the methods to the PinCodeActivity and allow accessing them publicly, out the of its regular life cycle.